front-end/src/components/List/ListItem/ListItem.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+
+import * as styles from './ListItem.scss';
+
+interface ListProps {
```

Comment:
```suggestion
interface ListItemProps {
```
This props interface belongs to the ListItem component so the name should correspond to the component's name.